iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonParser.h

```diff
@@ -0,0 +1,24 @@
+//
+//  MSMapGeoJsonParser.h
+//  MSMapsModules
+//
+//  Created by Elizabeth Bartusiak (t-elbart) on 6/17/20.
+//  Copyright © 2020 Microsoft Corporation.
+//  Licensed under the MIT license.
+//
+
+#import <Foundation/Foundation.h>
+#import <MicrosoftMaps/MicrosoftMaps.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSMapGeoJsonParser : NSObject
+
+- (MSMapGeoJsonParser *)init NS_UNAVAILABLE;
++ (MSMapGeoJsonParser *)new NS_UNAVAILABLE;
++ (nullable MSMapElementLayer *)parse:(nonnull NSString *)geojson
+                                error:(NSError * _Nullable *)error;
```

Comment:
This needs one more Nullable: NSError * nullable * nullable

Reply:
Use same set of annotations consistently: nullable instead of _Nullable.

Reply:
It seems that `nullable` is not permitted with double pointers. It could be either `NSError * _Nullable * _Nullable` or `NSError * __nullable * __nullable` (double underscores). Is there a preference for either one?

Reply:
_Nullable then please.